partner-chains-cli/src/config.rs

```diff
@@ -457,11 +464,20 @@ pub mod config_fields {
 		ConfigFieldDefinition {
 			config_file: RESOURCES_CONFIG_FILE_PATH,
 			path: &["cardano_payment_signing_key_file"],
-			name: "path to the payment signing file",
+			name: "path to the payment signing key file",
 			default: Some("payment.skey"),
 			_marker: PhantomData,
 		};
 
+	pub const CARDANO_COLD_VERIFICATION_KEY_FILE: ConfigFieldDefinition<'static, String> =
+		ConfigFieldDefinition {
+			config_file: RESOURCES_CONFIG_FILE_PATH,
+			path: &["cardano_cold_verification_key_file"],
+			name: "path to the cold verification key file",
+			default: Some("cold.vkey"),
```

Comment:
This is of course only a default value so users can pick whatever makes sense to them but `cold.vkey` seems very generic?

Reply:
Name of this key is arbitrary. I matches what Christos wonderful toolbox is using https://github.com/chrispalaskas/cardano-skepsis-toolbox/blob/1d3b00388b2307f8432b82b2f24fa915ee819e11/registerSPO.py#L117